samples/core/xgboost_training_cm/README.md

```diff
@@ -13,6 +13,9 @@ or not.
 ## Requirements
 
 Preprocessing uses Google Cloud DataProc. Therefore, you must enable the [DataProc API](https://cloud.google.com/endpoints/docs/openapi/enable-api) for the given GCP project.
+If KFP was deployed through K8S marketplace, please follow instructions in [the guideline](https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/guide.md#gcp-service-account-credentials)
```

Comment:
since you already mentioned Dataproc API in new text, you can delete the existing text.

Reply:
Done.